README.md

```diff
@@ -316,6 +317,19 @@ function renderSuggestion(suggestion) {
 
 **Important:** `renderSuggestion` must be a pure function (we optimize rendering performance based on this assumption).
 
+<a name="container-props-prop"></a>
+
+#### containerProps
```

Comment:
Just to make sure we remember: once the above is moved down somewhere, this'll need moving to match too